src/components/Footer/index.jsx

```diff
@@ -0,0 +1,135 @@
+import Image from "next/image"
+import Link from "next/link"
+
+export default function Footer() {
+  return (
+    <>
+      <footer>
+        <div className='relative lg:flex lg:justify-evenly bg-[#32314C] w-full h-full break-all mt-auto pb-32'>
+          {/* Logo Section */}
+          <div className='ml-5 pt-5'>
+            <Link href='/'>
+              <Image
+                src='logo-footer.svg'
+                width={350}
+                height={250}
+                alt='logo'
+              />
+            </Link>
+            <div className='absolute bottom-16 left-5 lg:left-40 lg:bottom-1'>
+              <a href='#'>
```

Comment:
Don't forget we work in NextJS framework so it is better to use the Link attribute in the next libraries !!